static/src/stylesheets/head.content.garnett.scss

```diff
@@ -47,3 +47,7 @@
 @import 'module/atoms/_youtube';
 @import 'module/atoms/_media';
 @import 'module/atoms/_explainers';
+
+
+// Temporary css added for a test of in-article audio players on Football plages
+@import 'module/journalism/_inline-audio-player.scss';
```

Comment:
Is this file the one loaded lazily?

Reply:
Good point, will remove and add to `_common.garnett.scss`

Reply:
Yeah, it should go in the lazyloaded CSS (just like the JS is loaded in `enhanced`), as this component is way below the fold.